waiter/src/waiter/util/client_tools.clj

```diff
@@ -1318,3 +1318,10 @@
          (map? jwt-config)
          (-> jwt-config :oidc-authorize-uri str/blank? not)
          (-> jwt-config :oidc-token-uri str/blank? not))))
+
+(defn exclusive-mode?
```

Comment:
It might be more clear if this was named `exclusive-by-default?` or something like that, since that's what it actually checks (defaults).

Reply:
Renamed function.